.github/workflows/node.js.yml

```diff
@@ -0,0 +1,29 @@
+# This workflow will do a clean install of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://help.github.com/actions/language-and-framework-guides/using-nodejs-with-github-actions
+
+name: Node.js CI
+
+on:
+  push:
+  pull_request:
```

Comment:
Do we want to be running build on push / pr? Might on release / tag be better?

Reply:
Especially pr, its a lot easier to review when you can just download the mod instead of cloning and building it

Reply:
I can see that use case now, I did not think about using the artifact for local testing.